KarhooSDK/Api/Response/Adyen/AdyenPaymentMethods/AdyenDetail.swift

```diff
@@ -5,42 +5,56 @@
 //  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
 //
 
+import Foundation
+
 public struct AdyenDetail: KarhooCodableModel {
+    public let confuguration: [String: String]
     public let items: [AdyenItem]
     public let key: String
     public let optional: Bool
     public let type: String
+    public let value: String
     
     public init(items: [AdyenItem] = [],
+                confuguration: [String: String] = [:],
                 key: String = "",
                 optional: Bool = false,
-                type: String = "") {
+                type: String = "",
+                value: String = "") {
         self.items = items
+        self.confuguration = confuguration
         self.key = key
         self.optional = optional
         self.type = type
+        self.value = value
     }
     
     enum CodingKeys: String, CodingKey {
+        case confuguration
         case items
         case key
         case optional
         case type
+        case value
     }
     
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.confuguration = (try? container.decode(Dictionary.self, forKey: .confuguration)) ?? [:]
         self.items = (try? container.decode(Array.self, forKey: .items)) ?? []
         self.key = (try? container.decode(String.self, forKey: .key)) ?? ""
         self.optional = (try? container.decode(Bool.self, forKey: .optional)) ?? false
         self.type = (try? container.decode(String.self, forKey: .type)) ?? ""
+        self.value = (try? container.decode(String.self, forKey: .value)) ?? ""
     }
-
+    
```

Comment:
this should have triggered a lint warning... 

make sure this is enabled https://twitter.com/johnsundell/status/952142890020540416?lang=en if it wasn't intentional 👍